packages/angular/src/generators/setup-mf/lib/add-cypress-workaround.ts

```diff
@@ -18,8 +17,10 @@ export function addCypressOnErrorWorkaround(tree: Tree, schema: Schema) {
   try {
     e2eProject = readProjectConfiguration(tree, e2eProjectName);
   } catch {
-    logger.warn(`Could not find an associated e2e project for ${schema.appName} with name ${e2eProjectName}. 
-    If the app does have an associated Cypress e2e project, you can pass the name of it to the generator using --e2eProjectName.`);
+    logger.warn(`Could not find an associated e2e project for ${schema.appName} with name ${e2eProjectName}.
+    If there is an associated e2e project for this application, and it uses Cypress, you will need to add a workaround to allow Cypress to test correctly.
```

Comment:
Should we maybe be a bit more explicit about which error is the workaround for? Maybe just mentioning what specific error requires the workaround.

Reply:
Yes probably a good idea